app/views/openid_connect/authorization/index.html.slim

```diff
@@ -0,0 +1,13 @@
+p = 'Allow app?'
+
+
+= button_to 'Allow',
+            openid_connect_allow_path,
+            params: @authorize_form.params,
+            class: 'btn btn-primary btn-wide'
+
+= button_to 'Decline',
+            openid_connect_deny_path,
+            method: :delete,
+            params: @authorize_form.params,
```

Comment:
Yes, I think it would make sense to provide an error back to the redirect_url that the client provides, so I want to make sure we have all the params so we can regenerate the errors